Demo/cql-demo.props

```diff
@@ -4,6 +4,10 @@
 		<TargetFrameworks>net8.0</TargetFrameworks>
 	</PropertyGroup>
 
+	<PropertyGroup>
```

Comment:
Maybe this property group can be joined with the one above?

Reply:
I kind of liked the grouping: compiler options (targetframework) vs. constants used as macros in our references....